mithril-aggregator/src/database/repository/cardano_transaction_repository.rs

```diff
@@ -413,7 +489,7 @@ mod tests {
         let cardano_transactions: Vec<CardanoTransactionRecord> = (20..=40)
             .map(|i| CardanoTransactionRecord {
                 transaction_hash: format!("tx-hash-{i}"),
-                block_number: i % 10,
+                block_number: i / 10,
```

Comment:
With the modulo the block number was cycling which made difficult the adaptation of this test.
It will be much easier to test anyway when we get rid of the dependency with the immutable number and work with the beacon number.